py/server/deephaven/query_library.py

```diff
@@ -0,0 +1,73 @@
+#
+#     Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+#
+"""This module allows users to import Java classes or packages into the query library for the Deephaven query engine.
+These classes or packages then can be used in Deephaven queries. """
+from typing import List
+
+import jpy
+from deephaven import DHError
+
+_JPackage = jpy.get_type("java.lang.Package")
+_JQueryLibrary = jpy.get_type("io.deephaven.engine.table.lang.QueryLibrary")
+_JClass = jpy.get_type("java.lang.Class")
+
+
+def import_class(name: str) -> None:
+    """Imports a Java class into the Query library so that it can be referenced in Deephaven queries. The class must
+    be reachable in the Deephaven server's classpath.
+
+    Args:
+        name (str): the full qualified name of the Java class
+
+    Raises:
+        DHError
+    """
+    try:
+        j_class = _JClass.forName(name)
+        _JQueryLibrary.importClass(j_class)
+    except Exception as e:
+        raise DHError(e, "failed to add the Java class to the Query Library.") from e
```

Comment:
Is this error message style "the way we do things"? (Meaning: lower case, ending in period?)

Reply:
yes, consistent throughout the API